instrumentation/executors/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/javaconcurrent/JavaExecutorInstrumentation.java

```diff
@@ -74,8 +74,8 @@
     @Advice.OnMethodEnter(suppress = Throwable.class)
     public static State enterJobSubmit(
         @Advice.Argument(value = 0, readOnly = false) Runnable task) {
-      Runnable newTask = RunnableWrapper.wrapIfNeeded(task);
-      if (ExecutorInstrumentationUtils.shouldAttachStateToTask(newTask)) {
+      if (ExecutorInstrumentationUtils.shouldAttachStateToTask(task)) {
+        Runnable newTask = RunnableWrapper.wrapIfNeeded(task);
```

Comment:
Could drop `newTask` and just assign to `task`.

Reply:
Done.